src/development/going-live.md

```diff
@@ -1,103 +1,89 @@
+3. If your domain is currently active elsewhere, the Time-To-Live (TTL) on your domain is set to the lowest possible value in order to minimize transition time.
+4. You have the auto-generated domain for your master branch.  This is the domain you see in the Location bar after selecting "Access site" in the UI.  You can also retrieve this value from the command line by running `platform environment:url` to see a list of all URLs that Platform.sh will serve for the current environment.  Write this down.
+5. Optional: If you want to guarantee that you have access to  your master environment before the domain name has switched over, use `ping` or any similar tool to determine the IP address of the master environment.  The IP address is not guaranteed stable but is unlikely to change during the course of the go-live process.
+
+## Set your domain
 
 First step is to [add your domain](/administration/web/configure-project.html#domains).
 
-You can add multiple domains to point to your project. Each domain can
-have its own SSL certificate.
+You can add multiple domains to point to your project. Each domain can have its own [SSL certificate](/development/going-live/ssl.md).
 
-After you have added your domain, your Master environment will no longer
-be accessible at `master-<project_id>.<cluster>.platform.sh`.
+After you have added your domain, your Master environment will no longer be accessible at `<environment>-<project>.<region>.platform.sh`.
 
-If you require access to the site, you can create a hosts file entry and
-point it to the IP address that resolves when you access your master
-project branch.
+If you require access to the site before the domain name becomes active you can create a hosts file entry and point it to the IP address that resolves when you access your master project branch.
 
 > **note**
 > If you are on a Development plan, you cannot add a domain. You would need to upgrade your subscription to a production plan.
 
-## 2 - Routes
 
-You can configure the routes of your project directly within the
-Web Interface or within your `.platform/routes.yaml` file.
+## Configure your DNS provider
 
-### Single hostname
+Configure your DNS provider to point your domain to your Platform.sh Master environment.
 
-If you want your site to only be available at `http://mydomain.com` and
-have `http://www.mydomain.com` redirect to `http://mydomain.com`, you
-need define your `routes.yaml` as follow:
+The way to do so will vary somewhat depending on your registrar, but nearly all registrars should allow you to set a CNAME.  Some will call it an Alias or similar alternate name, but either way the intent is to say "this domain should always resolve to... this other domain".  Add a CNAME record from your desired domain (`example.com`) to the master environment hostname you wrote down earlier.
```

Comment:
I see what you are trying to achieve here in terms of simplicity, however I think this will confuse anyone who actually understands CNAMEs, and their inability to be used on apex records. The instruction to add a CNAME from "example.com", for example, is both misleading and almost impossible. :)

I would suggest an alternative approach where we first provide advice on the apex domain, then on subdomains. Happy to take a shot at that.